### PR TITLE
should able to put the group according to the marathon api

### PR DESCRIPTION
**Pull request by sjkyspa**
#### delete /v2/groups/{group-id}?force={force}

image

### CODE REVIEW

**Comment by ssk2** [comment]
Nice, thank you!

**Comment by kensipe** [comment]
@sjkyspa could you fix the merge conflict?   It looks like this has been outstanding for some time... sorry about that... I'll get on it as soon as there is an update.

**Comment by sjkyspa** [comment]
@kensipe  I have fix the conflict, could you please merge the pull request?

File: src/main/java/mesosphere/marathon/client/Marathon.java [hunk 1]
```diff
@@ -78,15 +78,17 @@ DeleteAppTaskResponse deleteAppTask(@Param("app_id") String appId,
...
+	Result deleteGroup(@Param("id") String id, @Param("force") boolean force) throws MarathonException;
 	
 	@RequestLine("GET /v2/groups/{id}")
 	@Headers(HeaderUtils.MARATHON_API_SOURCE_HEADER)
 	Group getGroup(@Param("id") String id) throws MarathonException;
 
-	// Tasks
+	@RequestLine("PUT /v2/groups/{id}?force={force}")
+	@Headers(HeaderUtils.MARATHON_API_SOURCE_HEADER)
+	Result putGroup(@Param("id") String id, @Param("force") boolean force, Group group) throws MarathonException;
```
Comment (colin-msphere) [inline src/main/java/mesosphere/marathon/client/Marathon.java]: Please rename this to `updateGroup` to be consistent with the other `PUT` method.

**Comment by sjkyspa** [comment]
@colin-msphere putGroup have been to updateGroup.

**Pull request merged**